compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -1475,9 +1475,6 @@ class OMR_EXTENSIBLE CodeGenerator
    bool getSupportsArraySet() {return _flags1.testAny(SupportsArraySet);}
    void setSupportsArraySet() {_flags1.set(SupportsArraySet);}
 
-   bool getSupportsArraySetToZero() {return _flags3.testAny(SupportsArraySetToZero);}
```

Comment:
I agree that it's useful to distinguish; some JIT optimizations only care about "zero" arrayset support and it would be easier to get those optimizations working on a new platform if we did not have to have to implement the "full" arrayset support to begin with. I assume this was your reasoning as well @andrewcraik ?
 
@0dvictor Do you know what the status is on the ARM code generator re: arrayset support ?